spec/models/course/publish_enrichments_spec.rb

```diff
@@ -25,27 +26,40 @@
 
   describe "#content_status" do
     context "for a course without any enrichments" do
-      subject { create(:course, with_enrichments: []) }
+      subject { create(:course) }
       its(:content_status) { should eq(:empty) }
     end
 
     context "for a course an initial draft enrichments" do
-      subject { create(:course, with_enrichments: [[:initial_draft]]) }
+      let(:enrichments) { [build(:course_enrichment, :initial_draft)] }
+      let(:course) { create(:course, enrichments: enrichments) }
+      subject { course }
```

Comment:
This is now the same as what's on line 10-12 so can probably be removed.

Reply:
not quite, but yeah